source/common/buffer/buffer_impl.h

```diff
@@ -263,7 +262,7 @@ class OwnedSlice final : public Slice, public InlineStorage {
    */
   static SlicePtr create(uint64_t capacity) {
     uint64_t slice_capacity = sliceSize(capacity);
-    return SlicePtr(new (slice_capacity) OwnedSlice(slice_capacity));
+    return SlicePtr{new OwnedSlice(slice_capacity)};
```

Comment:
while you're touching it, `make_unique`?

Reply:
Not possible due to private constructor.